src/Uno.UI/UI/Xaml/Shapes/Shape.layout.cs

```diff
@@ -229,9 +229,8 @@ private protected Size MeasureAbsoluteShape(Size availableSize, NativePath? path
 			}
 
 			var stretch = Stretch;
-			var userSize = GetUserSizes();
-			var (userMinSize, userMaxSize) = GetMinMax(userSize);
-			var strokeThickness = StrokeThickness;
+			var stroke = Stroke;
+			var strokeThickness = stroke is null ? DefaultStrokeThicknessWhenNoStrokeDefined : StrokeThickness;
 			var pathBounds = GetPathBoundingBox(path); // The BoundingBox does also contains bezier anchors even if out of geometry
```

Comment:
@dr1rrb If this is referring to control points, then this comment is wrong. `GetPathBoundingBox` should return the bounds **not** including control points.

- Android already mentions that and has bugs due to that. https://github.com/unoplatform/uno/blob/4052a8d01827957b5045b85b7f0020483b4a6389/src/Uno.UI/UI/Xaml/Shapes/Shape.Android.cs#L169-L170
- Skia was also not using the correct path bounds which I fixed in this PR by using `TightBounds` instead of `Bounds`, and this fixed few cases already where the measuring was wrong.

Reply:
Android fix is https://github.com/unoplatform/uno/pull/13378

Reply:
Comment is now fixed.